src/vfxt/src/installvfxt.sh

```diff
@@ -234,6 +234,66 @@ function dump_env_vars() {
     echo "finish env dump"
 }
 
+function apt_get_update() {
+    retries=10
+    apt_update_output=/tmp/apt-get-update.out
+    for i in $(seq 1 $retries); do
+        timeout 120 apt-get update 2>&1 | tee $apt_update_output | grep -E "^([WE]:.*)|([eE]rr.*)$"
```

Comment:
set -e will cause a hard exit of the script when the grep doesn't find a match

Reply:
resolved